subjects/shoppinglistsort/README.md

```diff
@@ -4,7 +4,7 @@
 
 You were sent to the supermarket with a shopping list. To make your shopping faster, write a function `ShoppingListSort()` that takes a slice of strings and sorts it, according to the string length, returning a slice in which the strings appear in ascending order.
 
-When the length of string elements are equal, then the order in which they appear does not matter.
+- The length of the strings cannot repeat itself.
```

Comment:
Wouldn't it be better to say "The lengths of the strings being tested will be different". Feel free to resolve the conversation if you disagree.